driver/kubernetes/execconn/execconn.go

```diff
@@ -42,7 +43,8 @@ func ExecConn(restClient rest.Interface, restConfig *rest.Config, namespace, pod
 		remoteAddr: dummyAddr{network: "dummy", s: "dummy-1"},
 	}
 	go func() {
-		serr := exec.Stream(remotecommand.StreamOptions{
+		// FIXME(thaJeztah): should ExecConn get passed a context here? See https://github.com/kubernetes/kubernetes/pull/103177
+		serr := exec.StreamWithContext(context.TODO(), remotecommand.StreamOptions{
```

Comment:
Should `ExecConn` get passed the context, and punched through here? (Not too familiar with this code, and I'm horrible at contexts 😂)

Reply:
Yes, that's probably better